seqr/views/apis/project_api.py

```diff
@@ -197,6 +197,10 @@ def project_families(request, project_guid):
             JSONObject(paternalGuid='individual__father__guid', maternalGuid='individual__mother__guid'),
             filter=Q(individual__mother__isnull=False) | Q(individual__father__isnull=False), distinct=True,
         ),
+        phenotypePrioritizationTools=ArrayAgg(
```

Comment:
I added phenotypePrioritizationTools to two APIs -  project_families and details ([seqr/views/utils/orm_to_json_utils.py](https://github.com/broadinstitute/seqr/pull/4056/files#diff-91b87a6b0addc52510c5b65dadb7b810a7d4863b997db1ca751554de1f62f4cf) , but I also considered adding it as an attribute of a Sample instead so I could follow the `hasMatchingSampleFilter` pattern [here](https://github.com/broadinstitute/seqr/pull/4056/files#diff-8fbcc475f318e2396a365a6943ebb0a75e524d8099d133ea1c51b8447d450459R382) but it would make handling phenotypePrioritizationTools  on `loadedSamples` [here](https://github.com/broadinstitute/seqr/pull/4056/files#diff-4dc8b572f8d236793259125c7319e39372f2e269d33bbf87ac9ae403dc2aa1b3R151) weird. And conceptually, they're not really attributes of samples, they're attributes of individuals.